### PR TITLE
Make composer description prettier

### PR DESCRIPTION
**Pull request by techteam-seobility**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | yes
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

Using escapings like &gt; in the composer description makes it look bad on packagist https://packagist.org/packages/laminas/laminas-psr7bridge#1.3.x-dev

Replaced "<->" with "to" to improve display.

### CODE REVIEW

**Comment by techteam-seobility** [comment]
--signoff

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "laminas/laminas-psr7bridge",
-    "description": "PSR-7 &lt;-&gt; laminas-http message conversions",
+    "description": "PSR-7 to laminas-http message conversions",
```
Comment (weierophinney) [inline composer.json]:
```suggestion
    "description": "Bidirectional conversions between PSR-7 and laminas-http messages",
```

It's bi-directional, which is why we had `<->` previously. I'm suggesting something that makes that clear.

**Review by weierophinney — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (froschdesign) [reply composer.json]: The new description must also be added to:

https://github.com/laminas/laminas-psr7bridge/blob/746ced1704a521058d8d5bb3ee205fd947344622/mkdocs.yml#L8

Comment (techteam-seobility) [reply composer.json]: @froschdesign Does that mean it also has to be added here: https://github.com/laminas/laminas-psr7bridge/blob/1.3.x/docs/book/index.html or is this generated?

Comment (froschdesign) [reply composer.json]: @setete
No, this page is automatically generated and uses the description from the file `mkdocs.yml`. Therefore this file must also be updated.

**Comment by Ocramius** [comment]
Thanks @setete! Sorry it got unnoticed for so long :S

**Pull request merged**